src/main/scala/org/zalando/spearheads/innkeeper/routes/PostPaths.scala

```diff
@@ -36,9 +36,18 @@ class PostPaths @Inject() (
         team(authenticatedUser, token, "path") { team =>
           logger.debug(s"post /paths team $team")
 
-          hasOneOfTheScopes(authenticatedUser, reqDesc, scopes.WRITE) {
-            handleWith(savePath(UserName(authenticatedUser.username), TeamName(team.name), reqDesc))
-          }
+          (isAdminTeam(team, reqDesc) | hasOneOfTheScopes(authenticatedUser, reqDesc, scopes.ADMIN)) {
```

Comment:
these 2 statements should be combined in one directive, as we should aways use them together.